Packs/WhisperGateCVE-2021-32648/Playbooks/playbook-WhisperGate_&_CVE-2021-32648_6_5.yml

```diff
@@ -0,0 +1,2277 @@
+          "x": 1670,
+          "y": 1850
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "125":
+    id: "125"
+    taskid: df322ad1-d4c7-494b-8197-4a950f0f029a
+    type: regular
+    task:
+      id: df322ad1-d4c7-494b-8197-4a950f0f029a
+      version: -1
+      name: Collect Whispergate indicators from Unit 42
+      description: This script will extract indicators from given HTML and will handle bad top-level domains to avoid false positives caused by file extensions.
+      scriptName: ParseHTMLIndicators
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "8"
+    scriptarguments:
+      exclude_indicators:
+        simple: cdn.discordapp.com
+      unescape_domain:
+        simple: "True"
+      url:
+        simple: https://unit42.paloaltonetworks.com/ukraine-cyber-conflict-cve-2021-32648-whispergate/
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 2340,
+          "y": -1100
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "126":
+    id: "126"
+    taskid: 7e879edb-c319-4604-8f1d-acaf7cddc8e4
+    type: playbook
+    task:
+      id: 7e879edb-c319-4604-8f1d-acaf7cddc8e4
+      version: -1
+      name: Block Indicators - Generic v3
+      description: |+
+        This playbook blocks malicious indicators using all integrations that are enabled, using the following sub-playbooks:
+
+        - Block URL - Generic v2
+        - Block Account - Generic v2
+        - Block IP - Generic v3
+        - Block File - Generic v2
+        - Block Email - Generic
+        - Block Domain - Generic
+
+      playbookName: Block Indicators - Generic v3
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "42"
+    scriptarguments:
+      AutoBlockIndicators:
+        simple: "True"
+      AutoCommit:
+        simple: "No"
+      CustomBlockRule:
+        simple: "True"
+      CustomURLCategory:
+        simple: XSOAR Remediation - Malicious URLs
+      IP:
+        complex:
+          root: DBotScore
+          filters:
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: DBotScore.Type
+                iscontext: true
+              right:
+                value:
+                  simple: ip
+              ignorecase: true
+          - - operator: greaterThanOrEqual
+              left:
+                value:
+                  simple: DBotScore.Score
+                iscontext: true
+              right:
+                value:
+                  simple: "3"
+          accessor: Indicator
+          transformers:
+          - operator: uniq
+      MD5:
```

Comment:
Please ensure that the argument values on sub-playbook v3 match the argument values on the previous version of the sub-playbook Block Indicators - Generic v2. For example on v2 the value is ExtractedIndicators and here is DBotScore.Indicator